License2Deploy/rolling_deploy.py

```diff
@@ -123,24 +124,19 @@ def get_lb(self):
   def calculate_autoscale_desired_instance_count(self, group_name, desired_state):
     ''' Search via specific autoscale group name to return modified desired instance count '''
     try:
-      cur_count = int(self.get_group_info(group_name)[0].desired_capacity)
       if desired_state == 'increase':
-        new_count = self.double_autoscale_instance_count(cur_count)
+        self.old_desired_capacity = int(self.get_group_info(group_name)[0].desired_capacity)
+        self.new_desired_capacity = self.old_desired_capacity * 2
+        logging.info("Current desired count was changed from {0} to {1}".format(self.old_desired_capacity, self.new_desired_capacity))
       elif desired_state == 'decrease':
-        new_count = self.decrease_autoscale_instance_count(cur_count)
-      logging.info("Current desired count was changed from {0} to {1}".format(cur_count, new_count))
-      return new_count
+        logging.info("Current desired count was changed from {0} to {1}".format(self.new_desired_capacity, self.old_desired_capacity))
+        self.new_desired_capacity = self.old_desired_capacity
+      else:
+        raise Exception("Please make sure the desired_state is set to either increase or decrease")
```

Comment:
Again, this is to satisfy a test case.  We can change the test but it is really not related to the issue we are attacking here.